insights/parsers/satellite_yaml.py

```diff
@@ -0,0 +1,32 @@
+"""
+SatelliteYaml - file ``/etc/foreman-installer/scenarios.d/satellite.yaml``
+==========================================================================
+
+Parse the file ``/etc/foreman-installer/scenarios.d/satellite.yaml``.
+
+Sample input::
+
+    ---
+    :answer_file: /etc/foreman-installer/scenarios.d/satellite-answers.yaml
+    :installer_dir: /usr/share/foreman-installer/katello
+    :custom:
+        :lock_package_versions: true
+    :facts:
+        tuning: medium
+        mongo_cache_size: 6.25
+
+Examples:
+    >>> ':facts' in SatelliteYaml
+    True
+    >>> SatelliteYaml[':facts']['tuning']
+    'default'
```

Comment:
This result of the example is different from the above sample input. Could you please make them consistent?

Reply:
@xiangce, sure.